src/backend/vulkan/src/command.rs

```diff
@@ -836,30 +836,36 @@ impl com::CommandBuffer<Backend> for CommandBuffer {
             .cmd_draw_indexed_indirect(self.raw, buffer.raw, offset, draw_count, stride)
     }
 
-    unsafe fn draw_mesh_tasks(&mut self, _: TaskCount, _: TaskCount) {
-        unimplemented!()
+    unsafe fn draw_mesh_tasks(&mut self, task_count: TaskCount, first_task: TaskCount) {
+        self.device.mesh_fn.as_ref()
+            .expect("Draw command not supported. You must request feature MESH_SHADING_PIPELINE.")
```

Comment:
our feature(s) is named differently, right? We should use the proper name here, like `TASK_SHADER`

Reply:
Done.